### ISSUE TITLE
Accessibility: Narrator does not read contents in confirmation window

### ISSUE THREAD

**Issue by jithinjosepkl** [wixtoolset/issues#5993]
* Which version of WiX are you building with?

3.10.3.3010

* Which version of Visual Studio are you building with (if any)?

VS 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> (WiX Toolset Visual Studio Extension version in major.minor.release.build format)

* Which version of .NET are you building with?

4.0

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10

* Describe the problem and the steps to reproduce it.

During installation, if cancel is clicked - a new confirmation window is opened. Narrator does not read contents of this confirmation window. Attaching the video created by accessibility experts.
[3746290_A11y_MSMPI_MSMPISDKSetupPopup_Screenreader.zip](https://github.com/wixtoolset/issues/files/3139465/3746290_A11y_MSMPI_MSMPISDKSetupPopup_Screenreader.zip)

* Describe the behavior you expected and how it differed from the actual behavior.

> (Be as specific as you can and err on the side of providing too much information.)

## Feature requests

If this issue is a feature request:

* Describe the scenario and benefits that the feature supports.

> (Provide lots of specific detail so we understand the context around the feature.)

* Describe how you're accomplishing the feature today (if possible).

> (Be as specific as you can and err on the side of providing too much information.)

* Describe what you'd like the new feature to do.

> (There's no need to design it but suggestions are appreciated.)

**Comment by barnson** [comment]
Windows Installer is in charge of the UI at runtime. WiX is the build tool and cannot influence runtime behavior except as MSI allows. MSI would need to implement a change like this.
